GitHubRepositoryMerge.ps1

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+function Merge-GitHubRepositoryBranches
+{
+<#
+    .SYNOPSIS
+        Branch a branch into another
```

Comment:
I think this synopsis needs a bit more editing.

Reply:
Extremely sorry! that was a miss from my side. Thank you for the kind words overall to the contribution, that is much appreciated 👍